src/components/card/Description.vue

```diff
@@ -21,11 +21,22 @@
   -->
 
 <template>
-	<div>
+	<div @paste="onPaste">
 		<h5>
 			{{ t('deck', 'Description') }}
 			<span v-if="descriptionLastEdit && !descriptionSaving">{{ t('deck', '(Unsaved)') }}</span>
 			<span v-if="descriptionSaving">{{ t('deck', '(Saving…)') }}</span>
+			<span v-for="attachment in uploadQueue" :key="attachment.name" class="attachment">
+				<a class="fileicon" :style="mimetypeForAttachment()" />
```

Comment:
I guess this is copied from the attachments tab, so maybe you could use a separate component to share the code

Reply:
@juliushaertl I have make some updates.